camoco/cli/commands/health.py

```diff
@@ -155,11 +155,22 @@ def cob_health(args):
                 term.loci = list(filter(lambda x: x in cob, term.loci))
                 if len(term) < args.min_term_size or len(term) > args.max_term_size:
                     continue
+                #one tailed vs two tailed test
+                if args.two_tailed_GO is None:
+                    #run one tail for only positive values
+                    if cob.density(term.loci) > 0:
+                        density = cob.density(term.loci)
+                        density_emp.append(density)
+                    #skip negative density values
+                    else:
+                        continue
+                #if two_tailed_go is not none
+                else:
+                    density = cob.density(term.loci)
+                    density_emp.append(density)
                 term_ids.append(term.id)
                 term_sizes.append(len(term))
                 # ------ Density 
-                density = cob.density(term.loci)
-                density_emp.append(density)
                 # Calculate PVals
```

Comment:
We will also need to change the pval calculation depending if its one or two tailed. If it is two tailed, check the top 2.5% and bottom 2.5% instead of just the top 5%.